nservicebus/upgrades/absdatabus-3to4.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Azure Blob Storage Databus Upgrade Version 3 to 4
+reviewed: 2020-10-29
+component: ABSDataBus
+isUpgradeGuide: true
+---
+
+### SDK upgrade
+
+This version leverages improvements in the latest [Azure.Storage.Blobs](https://www.nuget.org/packages/Azure.Storage.Blobs) SDK, moving away from the [WindowsAzure.Storage](https://www.nuget.org/packages/WindowsAzure.Storage/) SDK.
+
+The `.BlockSize()` property has been deprecated due to restrictions of the new underlying SDK.
+
+### Improved retrieval
```

Comment:
Is that something for the upgrade guide? Sounds more like a thing that we would want to put into the announcement and blog post

Reply:
Also, I'm not sure what "improved retrieval" stands for.